src/gui/systray.cpp

```diff
@@ -193,6 +194,37 @@ void Systray::setPauseOnAllFoldersHelper(bool pause)
     }
 }
 
+void Systray::slotForceSyncAllFolders()
+{
+    // For some reason we get the raw pointer from Folder::accountState()
+    // that's why we need a list of raw pointers for the call to contains
+    // later on...
+    const auto accounts = [=] {
+        const auto ptrList = AccountManager::instance()->accounts();
+        auto result = QList<AccountState *>();
+        result.reserve(ptrList.size());
+        std::transform(std::cbegin(ptrList), std::cend(ptrList), std::back_inserter(result), [](const AccountStatePtr &account) {
+            return account.data();
+        });
+        return result;
+    }();
+    FolderMan *folderMan = FolderMan::instance();
+    const auto folders = folderMan->map();
+    for (auto f : folders) {
+        if (accounts.contains(f->accountState())) {
+            if (f->isSyncRunning()) {
+                f->slotTerminateSync();
+                folderMan->scheduleFolder(f);
+            }
+        }
+    }
+
+        // selectedFolder->slotWipeErrorBlacklist(); // issue #6757
```

Comment:
please remove those commented lines of code

Reply:
I'd love to, as soon as someone can comment on my initial question:

https://github.com/nextcloud/desktop/pull/3696#issue-716697469